.github/workflows/test.yml

```diff
@@ -91,10 +90,6 @@ jobs:
           echo \`\`\` >> $GITHUB_STEP_SUMMARY
       - name: Test
         run: ./mill -j0 firrtl.cross[].test -oF + svsim.cross[].test -oF + chisel[].test -oF
-      - name: Binary compatibility
-        # TODO either make this also check the plugin or decide that we don't
-        #      support binary compatibility for the plugin
-        run: sbt ++${{ inputs.scala }} unipublish/mimaReportBinaryIssues
 
```

Comment:
I don't have a replacement for this yet, there is `mill-mdoc` which looks like it was upgraded to mill 0.12 but hasn't been published yet--unclear but will try it out later. In nay case, this doesn't actually do anything on main as we don't have prior artifacts yet, so I think it's okay to drop temporarily.